src/rspec/spec/person_spec.rb

```diff
@@ -0,0 +1,66 @@
+require_relative '../files'
+require_relative '../../book'
+
+class Person < Nameable
+  attr_accessor :name, :age, :rentals, :parent_permission, :id
+
+  def initialize(age, name = 'unknown', parent_permission = true)
+    super()
+    @id = 25
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
```

Comment:
- [ ] Instead of redefining the class again, kindly use `require_relative` to make it available in this file 👍

Reply:
Noted and edited. Thanks for the feedback